web/docs/about-vast/README.md

```diff
@@ -0,0 +1,228 @@
+# About VAST
+
+:::tip Get Help
+Welcome to the VAST documentation! If you have any questions, do not hesitate to
+join our [community Slack](http://slack.tenzir.com) or open a [GitHub
+discussion](https://github.com/tenzir/vast/discussions).
+:::
+
+## What is VAST?
+
+VAST is an embeddable telemetry engine for structured event data, purpose-built
+for use cases in security operations. Consider VAST if you want to:
+
+- Build a federated detection and response architecture
+- Store, aggregate, and manage massive amounts of telemetry
+- Operationalize threat intelligence
+- Empower threat hunters
+- Engineer and deploy detections
+
+If you're unsure whether VAST is the right tool for your use case, keep reading.
```

Comment:
Only the first sentence explains what VAST is. The remainder explains why you'd want to use it. That doesn't match the section title.